README.md

```
@@ -1,4 +1,4 @@
 **Meu Portfólio**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)